index.html

```diff
@@ -477,7 +477,31 @@ <h3>Required Owned Elements</h3>
 		</section>
 		<section id="scope">
 			<h3>Required Context Role</h3>
-			<p>The required context role defines the owning container where this <a>role</a> is allowed. If a role has a required context, authors MUST ensure that an element with the role is contained inside (or <a>owned</a> by) an element with the required context role. For example, an element with role <code>listitem</code> is only meaningful when contained inside (or owned by) an element with role <code>list</code>.</p>
+			<p>The required context role defines the owning container where this <a>role</a> is allowed. If a role has a required context, authors MUST ensure that the element is contained inside (or <a>owned</a> by) an element with the required context role. There can be no other intermixed containing elements unless those elements are explicitly set as presentational, e.g. with a <rref>none</rref> role.</p>
+			<p>For example, an element with role <code>listitem</code> is only meaningful when contained inside (or owned by) an element with role <code>list</code>, as demonstrated by the following code snippet.</p>
+			<div class="example">
+				<pre class="highlight">
+          &lt;div role="list" aria-owns="owned_item">
+            &lt;div role="listitem">Direct child of the list.&lt;/div>
+            &lt;div role="none">
+              &lt;div role="listitem">Descendant contained by a presentational element.&lt;/div>
+            &lt;/div>
+          &lt;/div>
+          &lt;div role="listitem" id="owned_item">List item that is "owned" by its sibling "list".&lt;/div>
+        </pre>
+      </div>
+
+      <p>The following example shows <code>listitem</code> elements that do not meet the required context role expectations.</p>
```

Comment:
I don't like anti-patterns as examples. Can we put these 2 examples together and add comments to them to explain why they don't meet the requirements

Reply:
Fixed. Please see https://pr-preview.s3.amazonaws.com/w3c/aria/pull/1213.html#example-5